charts/aac-manage-case-assignment/values.preview.template.yaml

```diff
@@ -14,5 +14,5 @@ java:
         - notify-mca-api-key-secret
   environment:
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1260.service.core-compute-preview.internal
-    CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal
+    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1406.service.core-compute-preview.internal
+    CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api-pr-961.service.core-compute-preview.internal
```

Comment:
Should these URL really change?

Reply:
THis has been reverted in 3508db00c310df2fdce3145a61ab1698b59f4a48

Reply:
@MSancaktutar I have manually reverted the changes from #251 except for those made to `charts/aac-manage-case-assignment/values.yaml`.